storage/ndb/src/kernel/vm/Configuration.cpp

```diff
@@ -1275,7 +1273,8 @@ Uint64
 Configuration::compute_restore_memory()
 {
   Uint32 num_ldm_threads = globalData.ndbMtLqhWorkers;
-  Uint32 num_restore_threads = globalData.ndbMtRecoverThreads;
+  Uint32 num_restore_threads = 
```

Comment:
Could we directly use globalData.ndbMtQueryWorkers to compute instead? Since recover is the same with query workers.

Reply:
yep, simplified the code.